admin/availability/middleware.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const moment = require('moment')
+const config = require('../config')
+const checkWindowV2Service = require('../services/check-window-v2.service')
+
+async function isAdminWindowAvailable (req, res, next) {
+  if (config.OVERRIDE_AVAILABILITY_MIDDLEWARE) {
+    return next()
+  }
+  const currentDate = moment.utc()
+  const checkWindow = await checkWindowV2Service.getActiveCheckWindow()
+  if (!checkWindow || !currentDate.isBetween(checkWindow.adminStartDate, checkWindow.adminEndDate)) {
+    res.locals.pageTitle = 'Unavailable'
+    return res.render('availability/admin-window-unavailable', {
+      currentDate,
+      checkWindow,
+      startDateFormatted: checkWindow.adminStartDate.format('dddd D MMMM YYYY')
```

Comment:
What is this date format?
Is this a GDS approved one?

Reply:
Please use the date service to format dates, preferably using an existing known format.